docs/getting_started/tutorial.md

```diff
@@ -11,7 +11,7 @@ If you'd like to add Wagtail to an existing Django project instead, see [](integ
 
 Wagtail supports Python 3.7, 3.8, 3.9 and 3.10.
 
-To check whether you have an appropriate version of Python 3:
+To check the current version of python currently installed on your machine, run the command bellow:
```

Comment:
```suggestion
To check whether you have an appropriate version of Python 3:
```

@Dev-Yusuf I’m not necessarily opposed to changing this but I think it’s important we keep the wording for this relatively short.